.github/workflows/build.yml

```
@@ -6,6 +6,9 @@ on:
       - "releases/**"
     paths-ignore:
       - "**.md"
+    pull_request:
+      paths-ignore:
+        - "**.md"
```

Should be one indent less to make it work :smile: 

```suggestion
  pull_request:
    paths-ignore:
      - "**.md"
```

---

Oh, sorry! I'm not used to GitHub actions. I hope it's fine now.

I also removed the double quotes from original commit, for consistency with the rest of the CMakeLists.txt file.

---

Great, thanks!